guides/learning-and-development.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Learning and development
+---
+
+We have an annual learning and development allowance of £750 and up to 4 days
+per person.
+
+You can use this for professional and personal development activities like
+workshops, short courses or conferences.
+
+You can also use the allowance to attend a non role based conference or event
+focused on improving your health and wellbeing. You will receive two days off
+for this, and can top up the time from your annual leave allowance.
+
+We know that lots of training isn’t paid for, and involves things like coaching
+and mentoring, learning from colleagues, being an active part of networks, and
+other self-directed learning. We will encourage and support people who want to
+use unbilled days for different types of professional development.
+
+## How to use your learning and development allowance
+
+### Find an activity to book
+
+Finding the right thing to use the allowance for can be difficult. Here are some
+ways you can find ideas:
+
+- Check the
+  [Learning and development bikeshed post](https://bikeshed.dxw.com/?p=13876) to
+  see what other people have done this year
+- Ask other people on your team what they've done
+- Talk to your line manager, who can help you identify potential areas for
+  development and find suitable events or training
+
+### Book the activity
+
+- Agree the plan with your line manager before you book anything
+- Log it in [Breathe](https://hr.breathehr.com/) as a training request:
+  - Go to "My profile" on the top menu
+  - Click the "Training" button
+  - Click the "+" button to add a new request
+  - Include weblinks and an estimate of the ticket, accommodation and travel
+    costs (up to the maximum of £750, though you are welcome to pay the
+    difference for more expensive events)
+- Think about sharing event with the rest of your team if it's something other
+  people might find relevant/useful
+- Make sure your project team know when you're going to be away
```

Comment:
I think there is an additional step in breathe here. Apart from logging training. You also need to add 'Leave' request with the category training, so it updates the scheduling calendar.

Reply:
Perhaps add 'It also useful when you add your training days into your calendar for others to know' or something like that